atmos_model.F90

```diff
@@ -975,6 +975,7 @@ subroutine update_atmos_model_state (Atmos, rc)
 !--- local variables
   integer :: i, localrc, sec_lastfhzerofh
   integer :: isec, seconds, isec_fhzero
+  integer :: datm_temp
```

Comment:
maybe the previous name dtatm_temp is better? We have a data atm model called datm.